terraform/rancher/cluster/eks.tf

```diff
@@ -30,6 +30,9 @@ locals {
 module "eks_cluster" {
   source  = "terraform-aws-modules/eks/aws"
   version = "18.30.2"
+  # Switch off cloudwatch log group
+  create_cloudwatch_log_group = false
+  cluster_enabled_log_types = []
 
```

Comment:
Should we disable creation of cloudwatch log group for rancher cluster also?

Reply:
Well, I am not sure what are you meaning. I can't see any cloudwatch log group that we create in terraform code. Maybe I something miss, can you give me path for logs that you are mean ?